src/main/proto/io/dartinc/registry/v1beta1/registry.proto

```diff
@@ -27,3 +27,11 @@ message Controller {
   tech.figure.util.v1beta1.UUID                         controller_uuid = 1 [(validate.rules).message.required = true]; // ENote controller ID
   string                                                controller_name = 2 [(validate.rules).string.min_len = 1];      // ENote controller name
 }
+
+/**
+Metadata concerning how a collection of documents should be recorded on chain
+ */
+message DocumentRecordingGuidance {
+  reserved 1, 2;                               // Available for future use
+  map <string, bool> designated_documents = 3; // Markers for data
```

Comment:
what do the string and bool represent here?

Reply:
The string should be any identifier of a document — the idea is providing a generic model for further lightweight metadata, letting specific usecases interpret it however it pleases. There are multiple possibilities this allows for:
- Allowing a collection of document to be ingested more than once, but with different behaviors defined by this map
- Allowing for a subset of a collection of documents to be handled in a different manner (the motivation for this PR)
- etc.